Dockerfile

```diff
@@ -30,18 +33,19 @@ USER paneron
 WORKDIR ${project_path:?}
 
 COPY --chown=paneron:paneron package.json package.json
-COPY --chown=paneron:paneron yarn.lock yarn.lock
+COPY --chown=paneron:paneron pnpm-lock.yaml pnpm-lock.yaml
 # If you work on dependencies, like registry-kit or extension-kit
 # COPY --chown=paneron:paneron dependencies-local dependencies-local
-RUN yarn install
+RUN pnpm install
 
-RUN npx electron-rebuild
-# RUN yarn add typescript-language-server "typescript@4.2.2"
+# RUN pnpm --package=@electron/rebuild dlx electron-rebuild -v $(jq -r .devDependencies.electron < package.json)
 
 # see https://github.com/electron/electron/issues/17972
 USER root
-RUN chown root ${project_path:?}/node_modules/electron/dist/chrome-sandbox
-RUN chmod 4755 ${project_path:?}/node_modules/electron/dist/chrome-sandbox
```

Comment:
Per comments in the file, `Dockerfile` and `tsserver.Dockerfile` are intended to be line-by-line equivalent to aid subsequent refactoring. I’ll fix this.

Reply:
@strogonoff Thanks for looking into it.  This is the part that I got stuck in, having no idea how it's supposed to work in the first place.  Was trying to remove things that I thought might not be needed for various reasons...